internal/internal.go

```diff
@@ -26,10 +25,6 @@ func ConfigDir() (string, error) {
 		return filepath.Join(userConfigDir, "gops"), nil
 	}
 
-	if runtime.GOOS == "windows" {
-		return filepath.Join(os.Getenv("APPDATA"), "gops"), nil
-	}
-
 	if xdgConfigDir := os.Getenv("XDG_CONFIG_HOME"); xdgConfigDir != "" {
```

Comment:
This check is also redundant given the code in the cited section per ![image](https://user-images.githubusercontent.com/4898263/215287605-f547ecef-e7f9-4132-938c-eac2b12b1248.jpeg)

Reply:
Thanks! I've dropped the check for `XDG_CONFIG_HOME` as well.